platform/security/encryption/platform-security-encryption-impl/src/main/java/ddf/security/encryption/impl/EncryptionServiceImpl.java

```diff
@@ -114,6 +115,15 @@ public String decryptValue(String wrappedEncryptedValue) {
     return decrypt(encryptedValue);
   }
 
+  @Override
+  public String encryptValue(String unwrappedPlaintext) {
+    if (unwrappedPlaintext == null || unwrappedPlaintext.equals("")) {
```

Comment:
❓ Should we really be returning `null` if the string was empty? This doesn't seem in line with `decryptValue()`

Reply:
changed to be in line with decryptValue() for emptyString